### PR TITLE
feat(config): Aether Start

### LINKED ISSUE #1557: Feature: Start In Aether config for modpacks only
- Include disclaimer that other configuration would be necessary for this to not potentially mess with the player's ability to stay in the dimension without a portal.

### PR DESCRIPTION
**Pull request by bconlon1**
closes #1557

### CODE REVIEW

**Review by Jonathing — CHANGES_REQUESTED** [review]
Unless the user has Configured installed (whose GUI configs allow for default server configs on world creation), it will be very hard for a client, who wants to create a single player world, to have this config boolean magically enabled. It might be prudent to create a built-in datapack that simply triggers spawning the player in the Aether as well.

**Comment by bconlon1** [comment]
@Jonathing 
This config isn't intended for datapack makers and not really average players creating a world, as is said in the config comment. Either way though, this is still able to be set with `defaultconfigs` by anyone.

File: src/main/java/com/aetherteam/aether/capability/player/AetherPlayerCapability.java [hunk 1]
```diff
@@ -46,6 +46,7 @@ public class AetherPlayerCapability extends CapabilitySyncing implements AetherP
 	private static final UUID LIFE_SHARD_HEALTH_ID = UUID.fromString("E11710C8-4247-4CB6-B3B5-729CB34CFC1A");
 
 	private boolean canGetPortal = true;
+	private boolean canSpawnInAether = true;
```
Comment (bconlon1) [inline src/main/java/com/aetherteam/aether/capability/player/AetherPlayerCapability.java:49]: This value needs to be set to NBT.

**Comment by Jonathing** [comment]
Well, the modpack developers can figure it out. 😃👍

**Comment by bconlon1** [comment]
Meant to say *is intended for modpack players.

**Review by raptor494 — APPROVED** [review]

**Pull request merged**